File: phone_number_validation/java/Get.java
import java.net.http.HttpClient;
import java.net.http.HttpRequest;
import java.net.http.HttpResponse;
import java.io.IOException;
import java.net.URI;

public class Get {
  public static void main(String[] args) throws IOException {
    HttpClient client = HttpClient.newHttpClient();
    HttpRequest request = HttpRequest.newBuilder()
    .uri(URI.create("https://phonenumbervalidation.apifex.com/api/v1/validate?phonenumber=%2B1%20650-253-0000"))
    .build();

    client.sendAsync(request, HttpResponse.BodyHandlers.ofString())
    .thenApply(HttpResponse::body)
    .thenAccept(System.out::println)
    .join();
  }
}

File: phone_number_validation/nodejs/get.js
const https = require('https');

https.get('https://phonenumbervalidation.apifex.com/api/v1/validate?phonenumber=%2B1%20650-253-0000', (resp) => {
  let data = '';

  resp.on('data', (chunk) => {
    data += chunk;
  });

  resp.on('end', () => {
    console.log(JSON.parse(data)['is_valid_number']);
  });

});